jaeger-thrift/src/main/java/io/jaegertracing/thrift/internal/senders/ThriftSender.java

```diff
@@ -60,9 +60,9 @@ public int append(JaegerSpan span) throws SenderException {
     }
 
     byteBufferSize += spanSize;
-    if (byteBufferSize <= getMaxSpanBytes()) {
+    if (byteBufferSize <= getMaxBatchBytes()) {
       spanBuffer.add(thriftSpan);
-      if (byteBufferSize < getMaxSpanBytes()) {
+      if (byteBufferSize < getMaxBatchBytes()) {
```

Comment:
Renamed but the comparisons are the same as before. Since `byteBufferSize` contains process bytes, here we check whether it exceeds max batch size.

Reply:
I think this should continue comparing with getMaxSpanBytes, but we should remove the code that adds process size to byteBufferSize (in a couple of places)

Reply:
Yeah that would simplify the code overall. Will update soon